site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -104,25 +105,28 @@ export const WorkspacePage: FC = () => {
   const isLoading = !workspace || !template || !permissions;
 
   return (
-    <div css={{ height: "100%", display: "flex", flexDirection: "column" }}>
-      <Navbar />
-      {pageError ? (
-        <Margins>
-          <ErrorAlert
-            error={pageError}
-            css={{ marginTop: 16, marginBottom: 16 }}
+    <>
+      <ServiceBanner />
```

Comment:
I think the service banner should be included right before the `<Navbar />`

Reply:
Sure I can try that. I was emulating what the main layout was doing. But I can test both ways.

Reply:
Visually both ways works, so i go ahead the changes anyway. 

Here is how the service banner html looks like on all the pages using the `DashboardLayout.tsx`

<img width="788" alt="Screenshot 2024-03-01 at 8 48 52 AM" src="https://github.com/coder/coder/assets/5442469/cec90424-6337-4f9a-9bc7-f13c8cb9771d">

This current PR with changes to `WorkspacePage.tsx`

<img width="860" alt="Screenshot 2024-03-01 at 8 48 32 AM" src="https://github.com/coder/coder/assets/5442469/fee244ab-1b47-4f4e-bc6c-e6948519f000">

What it would look like with the proposed changes, as you can see the banner dev would be in a new location

<img width="570" alt="Screenshot 2024-03-01 at 8 53 08 AM" src="https://github.com/coder/coder/assets/5442469/63d3e156-4d67-47f9-b632-a81a3e58b809">

Reply:
Ahh I see, so we can leave it as it is. Thx for the changes.